app/apptesting/test_suite.go

```diff
@@ -0,0 +1,46 @@
+package apptesting
+
+import (
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	tmtypes "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	"github.com/regen-network/regen-ledger/v4/app"
+)
+
+type KeeperTestHelper struct {
+	suite.Suite
+
+	App         *app.RegenApp
+	Ctx         sdk.Context
+	QueryHelper *baseapp.QueryServiceTestHelper
+	TestAccs    []sdk.AccAddress
+}
```

Comment:
I realize this is what osmosis has but this seems more like an `AppTestSuite` than a `KeeperTestHelper`. I don't think we should copy over the naming for this.